main.py

```diff
@@ -1,12 +1,169 @@
+            postal_code = input("Enter the customer's new postal code: ")
+            for_update = video_store.update_customer(name=name, phone=phone, postal_code=postal_code)
+            mark_intro()
+            print("Here's the new account information: ", for_update) # Here's the new account information: {'id': 198, 'name': 'Maxine Waters', 
+                                                                        # 'phone': '333-333-0000', 'postal_code': '20022',
+                                                                        # 'registered_at': 'Sat, 29 May 2021 01:56:21 GMT', 'videos_checked_out_count': 0}
+        elif employee_choice == '5':
+            video_store.delete_customer()
+            mark_intro()
+            print("This customer account has been deleted. See below: ")
+            mark_intro()
+            for customer in video_store.list_customers():
+                print(customer)
+        elif employee_choice == '6':
+            mark_intro()
+            for video in video_store.list_videos():
+                print(video)
+        elif employee_choice == '7':
+            print("We'll add a new video to the database, then. ")
+            title = input("Enter the video's title: ")
+            release_date = input("Enter the video's release date (format: YYYY-MM-DD): ")
+            total_inventory = input("Enter the number of copies that'll be added to the stock: ")
+            available_inventory = total_inventory
+
+            for_display = video_store.create_video(title=title, release_date=release_date, total_inventory=total_inventory,\
+                                                    available_inventory=available_inventory)
+            mark_intro()
+            print("Video addition details: ", for_display) # {'id': 238}
+        elif employee_choice == '8':
+            choice_param = input("Want to select a video by title or ID? ")
+            if choice_param == "title":
+                title = input("Enter the title of the video you want to work with: ")
+                print("Video info you selected: ", video_store.get_video(title=title))
+            elif choice_param == "id": # address case issue later
+                cli_id = input("Which video ID would you like to select? ")
+                if (cli_id.isnumeric()) == False:
+                    print("Use digits, please.")
+                elif (cli_id.isnumeric()) == True:
+                    cli_id = int(cli_id)
+                    print("Video info you selected: ", video_store.get_video(id=cli_id)) # ... = {'id': 237, 'release_date': 'Sun, 12 May 2002 00:00:00 GMT', 
+                                                                                        # 'title': 'The Matrix', 'total_inventory': 5}
+            else:
+                print("Could not select. Please enter the title or ID of an existing video. ")
+        elif employee_choice == '9': # NOT WORKING 5/27/21, 8:27pm  
```

Comment:
I think you may have been struggling here because you misunderstood the requirements.

The goal was to update the video (the title, quantity, etc) and not to update a rental.

(I'm not sure updating rentals was supported by the API and would be much less straightforward if it was.)